src/reactA11yImgHasAltRule.ts

```diff
@@ -114,7 +118,8 @@ class ImgHasAltWalker extends Lint.RuleWalker {
             const allowNonEmptyAltWithRolePresentation: boolean = options.length > 1
                 ? options[1].allowNonEmptyAltWithRolePresentation
                 : false;
-
+            const imageFileNameRegex: RegExp = new RegExp('^.*\\.(jpg|bmp|jpeg|jfif|gif|png|tif|tiff)$', 'i');
```

Comment:
Seems odd to always recreate the regexp here each time. Could you please move this outside of the class?

Even if the underlying JS engine optimizes it away, it's a bit confusing reading through the code.